tests/webapp/api/test_push_api.py

```diff
@@ -300,6 +300,34 @@ def test_push_author(client, test_repository):
     assert set([result['id'] for result in results]) == set([1, 2])
 
 
+def test_push_reviewbot(client, test_repository):
```

Comment:
could you try doing like [here](https://github.com/mozilla/treeherder/pull/7416/files#diff-9106a24b9166c2c3b076b01263212cb02b481cd2b146f31173015fdafb15adc5R191) ? using @pytest.mark.parameterize